src/main/java/org/generationcp/middleware/dao/study/StudyEntrySearchDAO.java

```diff
@@ -280,6 +291,31 @@ private void addGuidScalar(final List<Scalar> scalars, final List<String> select
 		}
 	}
 
+	private void addBreedingMethodIdScalar(final List<Scalar> scalars, final List<String> selectClause,
+		final List<MeasurementVariable> entryDescriptors) {
+		if (!CollectionUtils.isEmpty(entryDescriptors)) {
+			// automatically add breeding method ID if method abbr is present (required for breeding method modal link)
```

Comment:
@cheryleigh can the modal be implemented using abbreviation if this is considered unique? so we keep this code simple and easy to maintain.

Reply:
hi @clarysabel , yup, that is indeed a better approach. I updated the modal to handle retrieval from abbreviation, thank you!